cvat-ui/src/components/requests-page/request-status.tsx

```diff
@@ -35,17 +35,21 @@ function StatusMessage(props: Props): JSX.Element {
     message = message || '';
     status = status || RQStatus.FINISHED;
     let textType: 'success' | 'danger' | 'warning' | undefined;
+    let classHelper: 'success' | 'failed' | 'queued' | undefined;
 
     if ([RQStatus.FAILED, RQStatus.UNKNOWN].includes(status)) {
         textType = 'danger';
+        classHelper = 'failed';
     } else if ([RQStatus.QUEUED].includes(status)) {
         textType = 'warning';
+        classHelper = 'queued';
     } else if ([RQStatus.FINISHED].includes(status)) {
         textType = 'success';
+        classHelper = 'success';
     }
     return (
         <Text
-            className='cvat-request-item-progress-message'
+            className={`cvat-request-item-progress-message cvat-request-item-progress-${classHelper}`}
```

Comment:
May we rewrite this piece of code without using `undefined`? 
E.g.

```ts
const [textType, classHelper] = ((_status: RQStatus) => {
    if (_status === RQStatus.FINISHED) {
        return ['success', 'success'];
    }

    if (_status === RQStatus.QUEUED) {
        return ['warning', 'queued'];
    }

    return ['danger', 'failed'];
})(status);
```

Reply:
We cant really go without `undefined` because the regular text (which should be displayed in case of `started` status) needs textType to be undefined

Reply:
Makes sense for textType, but in case of `started` status we will have `classHelper===undefined` and we will get className `cvat-request-item-progress-undefined`